contents/blog/posthog-changelog.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2023-02-24
+title: "The PostHog Changelog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+tags:
+  - Product updates
+  - Release notes
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Interested about what new features, fixes and updates have just launched on PostHog Cloud? Every Friday we update this page with a look back over the last seven days, so you can get a fresh look at what’s launched and launching right now. We also feature notable additions to the blog and docs. 
+
+> Some new features you read about below may still be in beta, behind a feature flag. Want to see which betas are currently available? [Check the roadmap](/roadmap)!
+
+## February 24, 2023
+
+### Performance monitoring in session recordings
```

Comment:
@joethreepwood Performance monitoring is a paid for feature, should we highlight this in this post?